csharp/src/elements/Geometry/Polygon.cs

```diff
@@ -91,19 +132,163 @@ public bool Contains(Polygon polygon)
         }
 
         /// <summary>
-        /// Tests if the supplied Polygon is within or coincident with the perimeter of this Polygon.
+        /// Tests if all Polygons in the supplied collection are within the perimeter of this Polygon.
         /// </summary>
-        /// <param name="polygon">The Polygon to compare.</param>
+        /// <param name="polygons">The collection of Polygons to compare.</param>
+        /// <returns>
+        /// Returns true if every supplied Polygon falls within the perimeter of this Polygon.
+        /// </returns>
+        public bool Contains(IList<Polygon> polygons)
+        {
+            var thisPath = this.ToClipperPath();
+            foreach (Polygon polygon in polygons)
+            {
+                var polyPath = polygon.ToClipperPath();
+                foreach (IntPoint vertex in polyPath)
+                {
+                    if (Clipper.PointInPolygon(vertex, thisPath) != 1)
+                    {
+                        return false;
+                    }
+                }
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// Tests if the supplied 2D Vector3 point is within or touches the perimeter of this Polygon.
```

Comment:
There's no such thing as a "2D Vector3". Do you mean to say "Tests whether a Vector3 in the same plane as the Polygon is within or touches..."?